Packs/CommonPlaybooks/Playbooks/playbook-Block_Email_-_Generic_v2.yml

```diff
@@ -0,0 +1,382 @@
+                            iscontext: true
+                          right:
+                            value:
+                              simple: active
+                    accessor: brand
+                iscontext: true
+    view: |-
+      {
+        "position": {
+          "x": 530,
+          "y": 120
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "5":
+    id: "5"
+    taskid: 5c5300a1-d7b2-41bc-80bb-1f768cf080e9
+    type: condition
+    task:
+      id: 5c5300a1-d7b2-41bc-80bb-1f768cf080e9
+      version: -1
+      name: Is FireEye Email Security (EX) Available?
+      description: Returns 'yes' if integration brand is available. Otherwise returns 'no'
+      scriptName: IsIntegrationAvailable
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "#default#":
+        - "2"
+      "yes":
+        - "9"
+    scriptarguments:
+      brandname:
+        simple: FireEye Email Security
+    results:
+      - brandInstances
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 950,
+          "y": 120
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "6":
+    id: "6"
+    taskid: 86892897-b219-45a4-8f41-f441ae27993d
+    type: condition
+    task:
+      id: 86892897-b219-45a4-8f41-f441ae27993d
+      version: -1
+      name: Is Mimecast v2 Available?
+      description: Returns 'yes' if integration brand is available. Otherwise returns 'no'
+      scriptName: IsIntegrationAvailable
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "#default#":
+        - "2"
+      "yes":
+        - "10"
+    scriptarguments:
+      brandname:
+        simple: MimecastV2
+    results:
+      - brandInstances
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 1350,
+          "y": 120
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "8":
+    id: "8"
+    taskid: 5012898a-1419-4604-8d37-2bbd60095da3
+    type: regular
+    task:
+      id: 5012898a-1419-4604-8d37-2bbd60095da3
+      version: -1
+      name: Cisco Email Security
+      description: Performs actions on list entries. Supports add, edit, and append.
+      script: "|||cisco-email-security-list-entry-add"
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      "#none#":
+        - "2"
+    scriptarguments:
+      action:
+        simple: append
+      list_type:
+        simple: blocklist
+      sender_addresses:
+        simple: ${inputs.EmailToBlock}
```

Comment:
Please use the following for the input:
![image](https://user-images.githubusercontent.com/28757135/183604716-d82644fc-92a8-4390-9038-646734e4314f.png)